### PR TITLE
[NewUI] Error handling in customize gas modal

### PR DESCRIPTION
**Pull request by danjm**
This PR adds error handling to the customize gas modal to ensure that the user cannot set a gas price or limit that they don't have the balance for, and to ensure that the limit cannot be set too low.

customgasmod-errors

### CODE REVIEW

File: ui/app/send-v2.js [hunk 1]
```diff
@@ -382,7 +373,14 @@ SendTransactionScreen.prototype.renderForm = function () {
...
 SendTransactionScreen.prototype.renderFooter = function () {
-  const { goHome, clearSend } = this.props
+  const {
+    goHome,
+    clearSend,
+    errors: { amount: amountError, to: toError }
+  } = this.props
+
+  const noErrors = amountError === null && toError === null
+  const errorClass = noErrors ? '' : '__disabled'
```
Comment (chikeichan) [inline ui/app/send-v2.js]: can you use the `classnames` lib?

File: ui/app/send-v2.js [hunk 2]
```diff
@@ -391,8 +389,8 @@ SendTransactionScreen.prototype.renderFooter = function () {
         goHome()
       },
     }, 'Cancel'),
-    h('button.send-v2__next-btn', {
-      onClick: event => this.onSubmit(event),
+    h(`button.send-v2__next-btn${errorClass}`, {
+      onClick: event => noErrors && this.onSubmit(event),
```
Comment (chikeichan) [inline ui/app/send-v2.js]: set `disabled` to true

File: ui/app/send-v2.js (hunk 1 above)
Comment (danjm) [reply ui/app/send-v2.js]: Done in 9fa9520b

File: ui/app/send-v2.js (hunk 2 above)
Comment (danjm) [reply ui/app/send-v2.js]: Done in 9fa9520b

**Pull request merged**